nix/t1/release/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, newScope
+
+, stdenvNoCC
+, fetchFromGitHub
+, runCommand
+, makeWrapper
+, typst
+, jq
+
+, configName
+, t1-script
+, ip-emulator
+, cases
+, elaborateConfigJson
+, pkgsCross
+}:
+
+lib.makeScope newScope (scope: rec {
+  inherit ip-emulator elaborateConfigJson;
+
+  testCases = with cases; [
+    intrinsic.matmul
+    intrinsic.softmax
+    intrinsic.linear_normalization
+  ];
+
+  emulator-wrapped = runCommand "ip-emulator"
+    {
+      nativeBuildInputs = [ makeWrapper ];
+    }
+    ''
+      mkdir -p $out/bin
+
+      makeWrapper ${t1-script}/bin/t1-helper $out/bin/ip-emulator \
+        --add-flags "ipemu" \
+        --add-flags "--config ${elaborateConfigJson}" \
+        --add-flags "--emulator-path ${ip-emulator}/bin/emulator"
+    '';
+
+  docker-layers = scope.callPackage ./docker-layers.nix {
+    rv32-gcc = pkgsCross.riscv32-embedded.buildPackages.gcc;
```

Comment:
Do we need musl or libc as well?

Reply:
I would love to use newlib, and modify the sbrk function to make malloc/free work on our new memory bank.

Reply:
ok wait for next PR. We may need some codegen for it